src/Crypto/RNCryptor/Types.hs

```diff
@@ -70,9 +73,21 @@ randomSaltIO :: Int -> IO ByteString
 randomSaltIO sz = C8.pack <$> forM [1 .. sz] (const $ randomRIO ('\NUL', '\255'))
 
 --------------------------------------------------------------------------------
+makeKey :: ByteString -> ByteString -> ByteString
+makeKey = generate (prfHMAC (undefined::SHA1)) (Parameters 10000 32) -- userKey salt
```

Comment:
curried args, I wanted to show user key comes first, then salt

Changed to make args explicit, e.g. \userKey salt -> ...